### PR TITLE
feat: Add a CLI to list and run ECS tasks

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?
Running a task on demand via the AWS web console is a bit of a faff, as you have to fill in a lot of fields, and it is easy to get wrong.

This change adds a CLI to make it easier to list and run tasks. There are three commands:
- `list-tasks` to  list all task definitions for a given set of `Stack`, `Stage`, and `App` tags
- `run-task` to run a single task using the task's `Stack`, `Stage`, `App` and `Name` tags (requires #253)
- `run-all-tasks` to run all tasks for a given set of `Stack`, `Stage`, and `App` tags

## Why?
Reduce human error in the AWS web console, and improve DX.

## How has it been verified?
Run the command(s) locally (from the root). For example:

```sh
npm run cli --workspace cloudquery list-tasks -- \
  --stack deploy \
  --stage CODE \
  --app service-catalogue
```

```sh
npm run cli --workspace cloudquery run-task -- \
  --stack deploy \
  --stage CODE \
  --app service-catalogue \
  --name DeployToolsListOrgs
```

### CODE REVIEW

File: packages/cloudquery/package.json [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
-    "start": "./script/start"
+    "start": "./script/start",
+    "cli": "ts-node src/index.ts"
+  },
+  "devDependencies": {
+    "@aws-sdk/client-ecs": "^3.398.0",
+    "@aws-sdk/client-ssm": "^3.398.0",
+    "@aws-sdk/credential-providers": "^3.398.0",
+    "@types/yargs": "^17.0.24",
+    "yargs": "^17.7.2"
```
Comment (akash1810) [inline packages/cloudquery/package.json:13]: Use `yargs` to get useful CLI things for free, such as usage instructions:

```console
➜ npm run cli --workspace cloudquery

> cloudquery@0.0.0 cli
> ts-node src/index.ts

index.ts COMMAND [args]

Commands:
  index.ts list-tasks     List all tasks for a given Stack, Stage and App
  index.ts run-task       Run a single task
  index.ts run-all-tasks  Run all tasks

Options:
      --version  Show version number                                   [boolean]
  -h, --help     Show help                                             [boolean]

```

File: packages/cloudquery/package.json [hunk 2]
```diff
@@ -2,6 +2,14 @@
   "name": "cloudquery",
```
Comment (akash1810) [inline packages/cloudquery/package.json:2]: This package name doesn't make sense anymore, as the package can run CloudQuery locally _and_ invoke ECS tasks. Maybe `dev` is a more accurate name? We can rename in a separate PR.

File: packages/cloudquery/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+const parseCommandLineArguments = () => {
+	return Promise.resolve(
+		yargs
+			.usage('$0 COMMAND [args]')
+			.command(
+				Commands.list,
+				'List all tasks for a given Stack, Stage and App',
+				(yargs) => {
+					yargs
+						.option('stack', {
```
Comment (NovemberTang) [inline packages/cloudquery/src/index.ts:25]: Could we choose defaults for some of these options?

**Review by tjsilver — APPROVED** [review]
👍

**Pull request merged**